plugin/src/main/resources/metadata/messages.xml

```diff
@@ -3799,4 +3799,45 @@ Path traversal <sup>[3][4]</sup> are not possible.
     </BugPattern>
     <BugCode abbrev="SECXSLT">A malicious XSLT could be provided</BugCode>
 
+    <!-- XSLT -->
```

Comment:
This comment that separate the description should be change to "Scala data leak" or something equivalent.